plugins/modules/keycloak_userprofile.py

```diff
@@ -406,7 +405,7 @@
   description: The data returned by the Keycloak API.
   returned: when state is present
   type: dict
-  sample: {'...': '...'}
+  sample: {"...": "..."}
```

Comment:
Since this sample isn't helpful at all, how about removing it completely?
```suggestion
```

Reply:
Agree